app/src/main/java/com/cvtracker/vmd/home/MainPresenter.kt

```diff
@@ -166,7 +164,6 @@ class MainPresenter(override val view: MainContract.View) : AbstractCenterPresen
     }
 
     override fun onSortChanged(sortType: SortType) {
-        selectedSortType = sortType
```

Comment:
I would keep the save of sortType here

Reply:
Ok, but this is not used by any other functions. We can retrieve from PrefUtil if needed somewhere later ?